tests/wpunit/API/V3/AuthorizerCacheTest.php

```diff
@@ -35,11 +36,11 @@ public function test_it_caches_a_valid_token_response(): void {
 		$this->assertTrue( $authorized );
 
 		// Cache should now be present.
-		$this->assertTrue( get_transient( $transient ) );
+		$this->assertTrue( ( 'true' === get_transient( $transient ) ) );
```

Comment:
```suggestion
		$this->assertSame( 1, get_transient( $transient )  );
```
Once you refactor with my example...